x/globalfee/types/params.go

```diff
@@ -45,6 +87,37 @@ func validateMinimumGasPrices(i interface{}) error {
 	return dec.Validate()
 }
 
+type BypassMinFeeMsgTypes []string
+
+// validateBypassMinFeeMsgTypes checks that bypass msg types aren't empty
+func validateBypassMinFeeMsgTypes(i interface{}) error {
+	bypassMinFeeMsgTypes, ok := i.([]string)
+	if !ok {
+		return sdkerrors.Wrapf(sdkerrors.ErrInvalidType, "type: %T, expected []sdk.Msg", i)
+	}
+
+	for _, msgType := range bypassMinFeeMsgTypes {
+		if msgType == "" {
+			return fmt.Errorf("invalid empty bypass msg type")
+		}
+
+		if !strings.HasPrefix(msgType, sdk.MsgTypeURL(nil)) {
```

Comment:
this is saying msg just begins with "/" is ok. 
the worst case is the setup of bypass-msg are not valid, ["/abc","/def"]
in the antehandler, it equals to nothing will bypass. and no err or damage right ?